plugins/sonar-core-plugin/src/main/resources/org/sonar/plugins/core/widgets/timeline.html.erb

```diff
@@ -53,9 +53,7 @@
          metric_data_map[metric_id].each() do |metric_data|
            # for every metric value, we need to check that the corresponding snapshot has values for each metric
            if metric_count_per_snapshot_id[metric_data[:sid]]==total_number_of_metrics
-             m_date = metric_data[:date]
-             # Only Oracle returns a Time object, so let's parse this string if it's not a Time instance
-             m_date = Time.parse(metric_data[:date]) unless m_date.is_a? Time
```

Comment:
metric_data[:date] is a long because metric_data_map is populated by TrendsChart.time_machine_measures (where we can see that the date is coming from snapshots.created_at).
When snapshots.created_at was a DateTime, Oracle was returning a Time object instead of a Date object, but as now this column is a long I think this 'hack' is useless.